runners/core-construction-java/build.gradle

```diff
@@ -57,8 +57,15 @@ task runExpansionService (type: JavaExec) {
   args = [project.findProperty("constructionService.port") ?: "8097"]
 }
 
-task testExpansionService(type: Jar) {
+task runTestExpansionService (type: JavaExec) {
+  main = "org.apache.beam.runners.core.construction.TestExpansionService"
+  classpath = sourceSets.test.runtimeClasspath
+  args = [project.findProperty("constructionService.port") ?: "8097"]
+}
+
+task jarTestExpansionService(type: Jar) {
```

Comment:
TestExpansionServiceJar might be a better name.

Reply:
Thanks for the suggestion. But doesn't it sound like a task for testing expansion service jar? (since conventionally the first word is considered as a verb). How about `assembleExpansionServiceJar` or `buildExpansionServiceJar`?

Reply:
+1 to BuildExpansionServiceJar.